### ISSUE TITLE
iex autocomplete on Windows

### ISSUE THREAD

**Issue by radar** [radar/joyofelixir#16]
@meddle0x53 talked to me on elixir-lang Slack and mentioned this:

> Hey, congratulations for the cool book. I've been teaching Elixir in the University here in Sofia, so I know that some of the people who read it will install Elixir under windows
> One problem they had was that the autocomplete and some other options were not working
> so you can add that they have to set the `IEX_WITH_WERL=true` env vaiable for windows like that, so they can get well-working `iex`
> maybe with a tutorial how to do it, hope this helps

I'll need to do some research to see if this is still a thing on Elixir even with the latest releases. If it is, then it's something that should _probably_ be added to the setup instructions for Windows in Appendix A.

**Comment by bodrovis** [comment]
I was interested in this feature on Windows as well and it does not seem to be working in Elixir 1.5.1. Running `iex --werl` does the trick but you end up with a separate erlang console in this case. As a side question, do you planning to translate the book into other languages? I can assists with Russian. Been really fond of Elixir lately.

**Comment by radar** [comment]
Thanks @bodrovis. Have you tried asking about this on elixir-lang/elixir? They might know.

I don't have any immediate plans on translating the book into other languages. My only plan is to finish it (eventually). Maybe after that I can start thinking about translating it into other languages.

**Comment by bodrovis** [comment]
No, I haven't as I don't really need this feature much. Just a quick tip for guys who really want it on Windows :)
